src/schema/v2/notifications/Item/__tests__/index.test.ts

```diff
@@ -176,7 +182,7 @@ describe("NotificationItem", () => {
                     "open edition",
                     "unique",
                   ],
-                  "internalID": "search-criteria-id",
+                  "internalID": "alert-id",
```

Comment:
This change in the response is what will ultimately fix the currently broken mutation in Eigen.